TARGET_VELOCITY: 1.388889
VELOCITY_RESOLUTION: 0.1
OMEGA_RESOLUTION: 0.1 
PREDICT_TIME: 5.0
TO_GOAL_COST_GAIN: 1.0
SPEED_COST_GAIN: 5.0
OBSTACLE_COST_GAIN: 10.0
HEAD_COST_GAIN: 1.0
ANGLE_RESOLUTION: 0.0087
MAX_DIST: 10.0
GOAL_THRESHOLD: 0.3
TURN_DIRECTION_THRESHOLD: 0.1